lib/web/ui/server.go

```diff
@@ -0,0 +1,61 @@
+package ui
+
+import (
+	"sort"
+
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// LabelDTO describes a label
```

Comment:
DTO is not clear to me, also it's redundant suffix here, not necessary.

Reply:
will change